fclones: add page (#12541)
tldr-pages/tldr: pages/common/fclones.md

# fclones

> Efficient duplicate file finder and remover.
> More information: <https://github.com/pkolaczk/fclones>.

> - Search a current directory:
> 
> `fclones group .`
> 
> - Search multiple directories and cache results:
> 
> `fclones group --cache {{directory1}} {{directory2}}`
> 
> - Search only the specified directory skipping the subdirectory and save the results into a file:
> 
> `fclones group {{path/to/directory}} --depth 1 > dupes.txt`
> 
> - Move the duplicates  in a `dupes.txt` to a different directory:
> 
> `fclones move {{path/to/target_directory}} <dupes.txt`

Reviewer: ```suggestion
- Search for duplicate files in the current directory:

`fclones group .`

- Search multiple directories for duplicate files and cache the results:

`fclones group --cache {{path/to/directory1 path/to/directory2}}`

- Search only the specified directory for duplicate files, skipping subdirectories and save the results into a file:

`fclones group {{path/to/directory}} --depth 1 > {{path/to/file.txt}}`

- Move the duplicate files in a TXT file to a different directory:

`fclones move {{path/to/target_directory}} < {{path/to/file.txt}}`
```

> - Perform a dry run for soft links  in a `dupes.txt` file without actually linking:
> 
> `fclones link --soft <dupes.txt --dry-run 2>/dev/null`

Reviewer: ```suggestion
- Perform a dry run for soft links  in a TXT file without actually linking:

`fclones link --soft < {{path/to/file.txt}} --dry-run 2 > /dev/null`
```

> - Perform a dry run for soft links  in a TXT file without actually linking:

Reviewer 2: ```suggestion
- Perform a dry run for soft links in a TXT file without actually linking:
```

- Delete the newest duplicates from the current directory without storing them in a file:

`fclones group . | fclones remove --priority newest`

> - Preprocess jpg files in the current directory by using an external command  to strip their exif data before matching for duplicates:

Reviewer: ```suggestion
- Preprocess JPG files in the current directory by using an external command  to strip their EXIF data before matching for duplicates:
```

> - Preprocess JPEG files in the current directory by using an external command  to strip their EXIF data before matching for duplicates:

Reviewer 2: ```suggestion
- Preprocess JPEG files in the current directory by using an external command to strip their EXIF data before matching for duplicates:
```
